assignments/prototypes.js

```diff
@@ -32,65 +52,49 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(statsChild) {
+  GameObject.call(this, statsChild);
+  CharacterStats.call(this, statsChild);
+  this.team = statsChild.team;
+  this.weapons = statsChild.weapons;
+  this.language = statsChild.language;
+}
+
+Humanoid.prototype = Object.create(GameObject.prototype);
+
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+
+Humanoid.prototype.greet = function() {
+  console.log(`${this.name} offers a greeting in ${this.language}`)
+}
  
 /*
   * Inheritance chain: GameObject -> CharacterStats -> Humanoid
   * Instances of Humanoid should have all of the same properties as CharacterStats and GameObject.
   * Instances of CharacterStats should have all of the same properties as GameObject.
 */
 
+const 
```

Comment:
Please remember to remove unnecessary lines and check your file for console errors before submitting. This line caused a console syntax error.